src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.tizen.cs

```diff
@@ -0,0 +1,64 @@
+using Microsoft.Maui.Platform;
+using NPointStateType = Tizen.NUI.PointStateType;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+public partial class MauiDrawingView : PlatformTouchGraphicsView
+{
+	/// <summary>
+	/// Initialize a new instance of <see cref="MauiDrawingView" />.
+	/// </summary>
+	public MauiDrawingView() : base(null)
+	{
+		previousPoint = new();
+		TouchEvent += OnTouch;
+	}
+
+	/// <inheritdoc />
+	protected override void Dispose(bool disposing)
+	{
+		if (disposing)
+		{
+			currentPath.Dispose();
+		}
+
+		base.Dispose(disposing);
+	}
+
+	/// <summary>
+	/// Initialize resources
+	/// </summary>
+	public void Initialize()
+	{
+		Drawable = new DrawingViewDrawable(this);
+		Lines.CollectionChanged += OnLinesCollectionChanged;
```

Comment:
do not forget to unsubscribe from the event

Reply:
It done by shared code 
https://github.com/CommunityToolkit/Maui/blob/e9b0a6c2f21bf49798e520e7c9442bdaba5ddfcd/src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.shared.cs#L74-L78

So no need to unsubscribe on here